src/library-authoring/studio-header/index.scss

```diff
@@ -57,26 +57,39 @@
   nav {
     .dropdown {
       .btn {
-        font-size: 1.6rem;
+        font-size: 1rem;
         font-weight: 600;
         background-color: white;
         color: $gray-d1;
         border-width: 1px;
-        line-height: 2.368rem;
+        line-height: 1.475rem;
+        padding: 0.234rem 0.469rem;
+        border-radius: ($baseline*0.1);
+        border-color: #f8f9fa;
```

Comment:
Please use variable for declare color palette

Reply:
added